### PR TITLE
Release `ibc-rs` v0.10.0

### PR DESCRIPTION
**Pull request by soareschen**
Closes: #1648

TODOs:
- [x] quickly re-check the guide (https://hermes.informal.systems)
- [x] As part of combing the guide, we should probably apply these changes https://github.com/informalsystems/ibc-rs/pull/1576/commits/2690ba8094cc15f15af5d6de195a2f207c4dec99
- [x] https://github.com/informalsystems/ibc-rs/pull/1664
- [x] Update the changed command flags in the guide (`--channel-version` and `-H`)
- [x] #1750
- [x] broken link in .changelog/unreleased/bug-fixes/1264-recover-acct-seq.md

## Description

- [x] Create a new release in the changelog, using [`unclog`](https://github.com/informalsystems/unclog).
- [x] Bump all crate versions to the new version.
- [x] Reassign unfinished issues of previous milestone to the next milestone.
- [x] Update Cargo.lock file (if re-publishing `ibc-relayer-cli`)

### CODE REVIEW

File: guide/src/config.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ For example, if there are only two chains configured, then Hermes will only rela
 i.e. the two chains will serve as a source for each other, and likewise as a destination for each other's relevant events.
 Hermes will ignore all events that pertain to chains which are unknown (ie. not present in config.toml).
 
-To restrict relaying on specific channels, or uni-directionally, you can use [packet filtering policies](https://github.com/informalsystems/ibc-rs/blob/v0.9.0/config.toml#L156-L173).
+To restrict relaying on specific channels, or uni-directionally, you can use [packet filtering policies](https://github.com/informalsystems/ibc-rs/blob/v0.10.0/config.toml#L156-L173).
```
Comment (hu55a1n1) [inline guide/src/config.md:39]: Looks like this link isn't pointing to packet-filtering policies.

File: guide/src/help.md [hunk 2]
```diff
@@ -74,10 +74,13 @@ FLAGS:
     --port-a PORT-A           identifier of the side `a` port for the new channel
     --port-b PORT-B           identifier of the side `b` port for the new channel
     -o, --order ORDER         the channel ordering, valid options 'unordered' (default) and 'ordered'
-    -v, --version VERSION     the version for the new channel
+    -v, --channel-version VERSION     the version for the new channel
 ```
 
-The `help` command is a replacement of the familiar `-h`/ `--help` flag typical for CLI applications.
+Additionally, the `-h`/`--help` flags typical for CLI applications work on
+most commands (with the exception of some commands currently using `-h` for
```
Comment (hu55a1n1) [inline guide/src/help.md:81]: Can we remove this comment now that other short options have been changed to `-H`?

**Review by hu55a1n1 — COMMENTED** [review]
Thanks, @soareschen and @romac! 👌 Just 2 minor comments from my side. 👍

**Comment by adizere** [comment]
To prevent confusion, will close this until we're ready (most likely in 2022).

**Comment by adizere** [comment]
Blocked by a potential regression documented in #1715. Once that is clarified, we resume this work.

File: guide/src/config.md (hunk 1 above)
Comment (romac) [inline guide/src/config.md:39]:
```suggestion
To restrict relaying on specific channels, or uni-directionally, you can use [packet filtering policies](https://github.com/informalsystems/ibc-rs/blob/v0.10.0/config.toml#L207-L224).
```

**Review by romac — APPROVED** [review]

**Pull request merged**